indico/modules/events/tracks/templates/display.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title %}
+    {{- page_title -}}
+{% endblock %}
+
+{% block page_actions %}
+    <div class="toolbar right">
+        <div class="group">
+            <a href="{{ url_for('tracks.program_pdf', event) }}"
+               class="i-button icon-file-pdf"
+               title="{% trans %}Download PDF{% endtrans %}"></a>
+        </div>
+    </div>
+{% endblock %}
+
+{% block content %}
+    {% if program %}
+        <div class="quotation programme">
```

Comment:
it's "program" internally (our code is not en_GB ;)) so the CSS class should be `program` too